src/libstd/rt/dwarf/mod.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! General DWARF format parsing utilities
```

Comment:
Similarly as above, it'd be great to have a lot more documentation on this module. For example the `read_uleb128` function would be nice to have a hyperlink to the definition of the format, and some overall docs about what this module's doing would be nice.